process-dataset/data-validation/schema.js

```diff
@@ -120,8 +172,8 @@ const manifestSchema = {
         "thumbnailRoot",
         "downloadRoot",
         "volumeViewerDataRoot",
-        "defaultXAxis",
-        "defaultYAxis",
+        "xAxis",
+        "yAxis",
```

Comment:
I'm probably missing something here but is `groupBy` not required?

Reply:
- [ ] make colorBy required too